src/RTCDTMFSender.ts

```diff
@@ -0,0 +1,55 @@
+import { NativeModules } from 'react-native';
+
+const { WebRTCModule } = NativeModules;
+const DEFAULT_DURATION = 300;
+const DEFAULT_INTER_TONE_GAP = 70;
+const MIN_DURATION = 70;
+const MAX_DURATION = 6000;
+const MIN_INTER_TONE_GAP = 50;
+
+export default class RTCDTMFSender {
+    _peerConnectionId: number;
+    _senderId: string;
+    constructor(peerConnectionId: number, senderId: string) {
+        this._peerConnectionId = peerConnectionId;
+        this._senderId = senderId;
+    }
+
+    get canInsertDTMF(): boolean {
+        return WebRTCModule.senderCanInsertDTMF(
+            this._peerConnectionId,
+            this._senderId
+        );
+    }
+
+    get toneBuffer(): string {
+        return WebRTCModule.getDTMFToneBuffer(
+            this._peerConnectionId,
+            this._senderId
+        );
+    }
+
+    insertDTMF(
+        tones: string,
+        duration = DEFAULT_DURATION,
+        interToneGap = DEFAULT_INTER_TONE_GAP
+    ) {
+        if (duration < MIN_DURATION) {
```

Comment:
I think you can rewrite this like so: `Math.min(Math.max(duration, MIN_DURATION), MAX_DURATION)`